medical_prescription_us/models/medical_prescription_order_line.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from dateutil.relativedelta import relativedelta
+from openerp.exceptions import ValidationError
+
+
+from openerp import api, fields, models, _
+
+
+# @TODO: Abstract control codes into core, add months till expire
```

Comment:
Move this to roadmap. Link to RFC https://github.com/OCA/product-attribute/issues/225